eel/eel-editable-label.c

```diff
@@ -1349,9 +1353,21 @@ get_layout_location (EelEditableLabel  *label,
     misc = GTK_MISC (label);
 #endif
     widget = GTK_WIDGET (label);
-#if !GTK_CHECK_VERSION (3, 0, 0)
+#if GTK_CHECK_VERSION (3, 0, 0)
+    margin_start = gtk_widget_get_margin_start (widget);
+    margin_end = gtk_widget_get_margin_end (widget);
+    margin_top = gtk_widget_get_margin_top (widget);
+    margin_bottom = gtk_widget_get_margin_bottom (widget);
+    gtk_widget_get_halign (widget);
```

Comment:
Yes, I'll check it later. I suggest moving all changes to ```eel/eel-editable-label.c``` to another branch and merge PR w/o them (as other changes seem ok).